docs/Developers/Newcomers_Guide.rst

```diff
@@ -392,6 +392,18 @@ Step 4. Sending Your Changes
    Your current branch will have an asterisk (\*) next to it. Ensure that there
    is no asterisk next to the master branch.
 
+Add your changes:
+
+::
+
+    $ git add .
```

Comment:
This is prone to error. Very likely, newcomers add some random files to the repository, and by using `git add .` they add random files and commit.